demo/site/package.json

```diff
@@ -16,7 +16,7 @@
         "generate-block-types:watch": "chokidar -s \"**/block-meta.json\" -c \"npm run generate-block-types\"",
         "gql:types": "graphql-codegen",
         "gql:watch": "graphql-codegen --watch",
-        "lint": "run-p gql:types generate-block-types && run-p lint:prettier lint:eslint lint:tsc",
+        "lint": "run-s && run-p gql:types generate-block-types && run-p lint:prettier lint:eslint lint:tsc",
```

Comment:
the empty run-s makes no sense and leads to following error when executing:

<img width="1050" height="752" alt="Image" src="https://github.com/user-attachments/assets/ebd8eacb-5f90-441d-9d8b-cc5dc74e236c" />

```suggestion
        "lint": "run-p gql:types generate-block-types && run-p lint:prettier lint:eslint lint:tsc",
```